doc/tutorial_conclusions.rst

```diff
@@ -213,7 +213,11 @@ Here follows the source code and configuration used in the examples.
        },
        "disk":
        {
-           "image": "/home/username/images/IE8_-_Win7-disk1.qcow2",
+           "image":
+           {
+               "uri": "/home/username/images/IE8_-_Win7-disk1.qcow2",
+               "provider": "see.image_providers.DummyProvider"
+           }
```

Comment:
The dummy provider will work exactly the same as the old syntax. if `"image": "/absolute/path"` used to work, so will this.